packages/pn-personafisica-webapp/src/pages/NuovaDelega.page.tsx

```diff
@@ -152,6 +152,17 @@ const NuovaDelega = () => {
     dispatch(resetNewDelegation());
   }, []);
 
+  const deleteInput = (
+    funField: (field: string, setValue: any, validation: boolean | undefined) => void,
+    funTouched: (field: string, setValue: boolean, validation: boolean) => void
+  ): MouseEventHandler<HTMLLabelElement> => {
```

Comment:
Maybe it is not necessary the return

Reply:
if i delete, return an error with the type of function MouseEventHandler

<img width="929" alt="image" src="https://user-images.githubusercontent.com/119851973/209365826-ed734021-7aea-40d8-bbaf-188dae3692d2.png">

Reply:
@leleOFA , @AndreaCimini90 is right, you can just change the return type of `deleteInput` to `void` and delete the `return` statement.